llvm/test/Transforms/InstCombine/trunc-extractelement-spir.ll

```diff
@@ -5,8 +5,8 @@ target triple = "spir"
 
 define i32 @shrinkExtractElt_i64_to_i32_0(<3 x i64> %x) {
 ; CHECK-LABEL: @shrinkExtractElt_i64_to_i32_0(
-; CHECK-NOT:    [[TMP1:%.*]] = bitcast <3 x i64> [[X:%.*]] to <6 x i32>
-; CHECK-NOT:    [[T:%.*]] = extractelement <6 x i32> [[TMP1]], i32 0
+; CHECK-NOT:    {{%.*}} = bitcast <3 x i64> [[X:%.*]] to <6 x i32>
+; CHECK-NOT:    [[T:%.*]] = extractelement <6 x i32> {{%.*}}, i32 0
```

Comment:
Probably `X` and `T` are not needed as well.

Could you clarify the reasons for this change?

Reply:
There was a fix to FileCheck in LLORG (fd941036bf249) - now it's not allowed to use variables inside `CHECK-NOT` checks.